### PR TITLE
Set ffmpeg root when building ffmpeg

### PR DESCRIPTION
**Pull request by aashish24**

### CODE REVIEW

**Comment by aashish24** [comment]
Ref: https://github.com/UV-CDAT/uvcdat/issues/1639

**Comment by aashish24** [comment]
Requires this to be merged first: https://github.com/UV-CDAT/VTK/pull/12

**Comment by aashish24** [comment]
@dnadeau4 @sankhesh @doutriaux1

**Comment by doutriaux1** [comment]
@aashish24 both PR look good to me. But in order for the bots to pick this up we will need the VTK to be merged first. And @dnadeau4 has questions for you on UV-CDAT/VTK#12

**Comment by aashish24** [comment]
thanks @doutriaux1, I pushed a fix for the issue @dnadeau4 referred. It should be good now in VTK.

**Comment by doutriaux1** [comment]
ok once these are done I will retrigger

**Comment by aashish24** [comment]
FYI: @doutriaux1 @dnadeau4 merged my VTK branch

**Comment by dnadeau4** [comment]
Let see if it passes.

**Comment by doutriaux1** [comment]
ok relaunched the request

**Comment by aashish24** [comment]
@doutriaux1 looks like is good

**Comment by aashish24** [comment]
@dnadeau4 it would be great if you can try this on the machine where you had the issue.

**Comment by aashish24** [comment]
@doutriaux1 I updated the branch with current master, once bots are done, please review.

**Comment by doutriaux1** [comment]
thx!

**Comment by aashish24** [comment]
@doutriaux1 ?

**Comment by aashish24** [comment]
@doutriaux1 looks good to me..

**Comment by aashish24** [comment]
I meant..the dashboard looks good to me..

**Comment by aashish24** [comment]
@doutriaux1 ping!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-uvcdat
+uv-cdat
```
Comment (sankhesh) [inline README.md]: Shouldn't this be `UV-CDAT`?

Comment (aashish24) [inline README.md]: > Shouldn't this be UV-CDAT?

this is not from my commit but rather came from Charle's merge. I am suspecting this is what we have in the master.

Comment (aashish24) [inline README.md]: I think @doutriaux1 did it to trigger build.. I will delete it.. thanks for finding it.

**Pull request merged**